modules/ROOT/pages/managing-api-clients.adoc

```diff
@@ -3,16 +3,36 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic provides an overview of client management on Anypoint Platform. As an API owner, you can apply an OAuth 2.0 policy to authenticate client applications that try to access your API. You need an OAuth provider to use an OAuth 2.0 policy. One of the following OAuth providers is recommended:
+This topic provides an overview of client management on Anypoint Platform. As an API owner, you can apply an OAuth 2.0 policy to authenticate client applications that try to access your API. You need an OAuth provider to use an OAuth 2.0 policy.
 
-* OpenAM version: 14
-* PingFederate versions: 6 - 8
-* Dynamic Client Registration-compliant identity providers
+Using the client management, you can configure several client providers to manage client credentials.
+Any client providers you configure are available in your organization.
+You can make them available to business group administrators to xref:environments.adoc#add-an-idp-to-an-existing-environment[associate the client providers with different environments] depending on your organization's policies.
+When you delete a client provider from your master organization, the client provider is no longer available in environments that used it, and assets that depended on the client provider may break.
```

Comment:
I think we need to characterize "break" here.

Reply:
@sebastiankorol @santro92 What details should I include here?